packages/api/src/services/emp-stats.ts

```diff
@@ -139,9 +139,46 @@ export default (config: Config, appState: Dependencies) => {
     });
   }
 
+  async function backfillTvl(emp: uma.tables.emps.Data, priceSample: PriceSample) {
+    assert(emp.collateralCurrency, "Backfill TVL Requires collateral currency for emp: " + emp.address);
+    const value = await calcTvl(priceSample[1], emp).toString();
+    return getTvlHistoryTable().create({
+      address: emp.address,
+      timestamp: priceSample[0],
+      value,
+    });
+  }
+
+  async function backfillAllTvl(addresses: string[]) {
+    return Promise.allSettled(
+      addresses.map(async (address) => {
+        const emp = await getFullEmpState(address);
+        assert(uma.utils.exists(emp.collateralCurrency), "Emp has no collateral currency: " + emp.address);
```

Comment:
does this mean if one of the addresses in this call do not exist the whole addition will fail? that might be ok, just wanted to point it out